src/main/java/specs/RestAssuredSpec.java

```diff
@@ -0,0 +1,22 @@
+package specs;
+
+import io.restassured.builder.ResponseSpecBuilder;
+import io.restassured.http.ContentType;
+import io.restassured.specification.RequestSpecification;
+import io.restassured.specification.ResponseSpecification;
+
+import static helpers.ConfigSetUp.getHost;
+import static helpers.ConfigSetUp.getToken;
+import static io.restassured.RestAssured.with;
+
+public class RestAssuredSpec {
+    static public RequestSpecification requestSpecification =
+            with()
+                    .baseUri(getHost())
+                    .headers("Authorization", "bearer " + getToken())
```

Comment:
Could be replaced with org.apache.http.HttpHeaders.AUTHORIZATION.

Reply:
replaced